theme/src/main/resources/com/github/weisj/darklaf/theme/high_contrast_light/high_contrast_light_defaults.properties

```diff
@@ -83,6 +83,7 @@ Theme.highContrast                = true
 %widgetFillDefault                = #00D1E5
 
 ####Controls####
+%control                          = #FFFFFF
```

Comment:
I don’t like that this duplicates the `background` property in every theme. I’d suggest to instead set the property in [ThemeDefaultsInitTask](https://github.com/weisJ/darklaf/blob/master/core/src/main/java/com/github/weisj/darklaf/task/ThemeDefaultsInitTask.java) based on the value of `background`.

Reply:
You are right — this is a preferable solution indeed. But, in order to avoid `background` property duplication, what about creating a global reference to `background` property in **/darklaf/core/src/main/resources/com/github/weisj/darklaf/globals.properties**? 
![Schermata 2022-01-16 alle 05 34 10](https://user-images.githubusercontent.com/97770968/149647389-af54112b-ff16-4c08-89be-1a90c6d97cd4.png)
Or, just for clarity's sake, a better approach would be to create ThemeDefaultsInitTask#initControl method for initialising the `control` property?
Thanks in advance.

Reply:
That’s actually the better solution :). Maybe add a comment explaining why it’s needed as well.